src/cmd/ksh93/sh/path.c

```diff
@@ -1613,8 +1613,13 @@ void path_alias(Namval_t *np, Pathcomp_t *pp) {
         Shell_t *shp = sh_ptr(np);
         struct stat statb;
         char *sp;
+        Pathcomp_t *old = 0;
```

Comment:
s/0/NULL/ please (Yes, I know the existing code uses zero liberally where NULL makes the intent clearer but no sense adding more bad idioms :-)

Reply:
Also, no need to initialize it here since it isn't used before you assign to it below.